zebra-chain/src/orchard/arbitrary.rs

```diff
@@ -18,14 +16,14 @@ impl Arbitrary for Action {
     fn arbitrary_with(_args: Self::Parameters) -> Self::Strategy {
         (
             any::<note::Nullifier>(),
-            any::<VerificationKeyBytes<SpendAuth>>(),
+            any::<SpendAuthVerificationKeyBytes>(),
             any::<note::EncryptedNote>(),
             any::<note::WrappedNoteKey>(),
         )
             .prop_map(|(nullifier, rk, enc_ciphertext, out_ciphertext)| Self {
                 cv: ValueCommitment(pallas::Affine::identity()),
                 nullifier,
-                rk,
+                rk: rk.0,
```

Comment:
Hm i wonder if we can make this nicer, but not a blocker

Reply:
Meh ignore

Reply:
It has a deref impl, so `&rk` might also work, either as an argument or in the struct initialisation.